### ISSUE TITLE
Make "Call for Papers" embedabble 

### ISSUE THREAD

**Issue by saschafoerster** [pretalx/pretalx#980]
I would like to embed not only the programm but also the Call for Paper into an external website, for instance venueless.org. 
So people with no ticket could add some content to a conference in the same website they will use for the conference. It also allows for barcamps to keep people in venueless itself, when they are adding some sessions: 

I tried to create a simple website with the widget-code to add it in an iframe, but this is only possible for the programm right now.

**Comment by rixx** [comment]
Yes, this is currently not possible and a rather large endeavour. Making things embeddable scales terribly with the pretalx layout, so as with the program widget, we would go with a JavaScript version of our CfP flow.

But since organisers can configure the CfP in a ton of different ways, replicating all that logic in JavaScript will be costly – especially to keep it synced after the fact. Every time we add a new question type, we have to add it both in regular pretalx and the pretalx widget, making every further change to pretalx harder and less likely.

I'd be very happy to guide a PR on this, but unless this feature gets sponsoring/support, I won't be able to prioritise it myself.

**Comment by saschafoerster** [comment]
@rixx How about a config-field, that allows the server-administrator to embed pretalx via iframe only in some certain defined domains? This is what I know from other tools. Or to allow to turn X-Frames in pretalx off and control it myself with the nginx-proxy myself? Too insecure?
